cpu/arm7_common/Makefile.dep

```diff
@@ -1,7 +1,7 @@
 # use common ARM7 periph code
 USEMODULE += arm7_common_periph
 
-ifeq (1,$(PICOLIBC))
+ifneq (,$(filter picolibc,$(FEATURES_OPTIONAL) $(FEATURES_REQUIRED)))
```

Comment:
I was afraid Travis would not like this :(

But I can't check `$(FEATURES_USED)` as in the first go-around picolibc is not in the used list yet, so newlib gets selected.

Reply:
Hmm that seems weird and wrong. Let me check.

Reply:
(wrong in the sense that the handligng is different in `info-global.inc.mk` and in `dependency_resolution.inc.mk`)

Reply:
See #15013.